packages/ffe-cards/less/common-card-styling.less

```diff
@@ -17,33 +17,67 @@
         }
     }
 
-    &:is(a, button, .ffe-card-base--clickable) {
+    button.ffe-card__action--raw {
+        background: none;
+        color: inherit;
+        border: none;
+        padding: 0;
+        font: inherit;
         cursor: pointer;
-        &:visited {
-            .native & {
-                @media (prefers-color-scheme: dark) {
-                    color: var(--ffe-farge-hvit);
-                }
-            }
-        }
+        outline: inherit;
+    }
 
-        &:focus {
-            box-shadow: 0 0 0 2px var(--ffe-g-primary-color);
-        }
+    a.ffe-card__action--raw {
+        color: inherit;
+        text-decoration: inherit;
+        outline: inherit;
+    }
 
-        &:focus,
-        &:active {
-            border-color: var(--ffe-g-primary-color);
-        }
+    &:has(.ffe-card__action) {
+        cursor: pointer;
 
         @media (hover: hover) and (pointer: fine) {
             &:hover {
                 border-color: var(--ffe-g-primary-color);
+                &
+                    :where(
+                        .ffe-card-body__card-name,
+                        .ffe-card-body__title,
+                        .ffe-card-body__subtext,
+                        .ffe-card-body__text
+                    ):has(.ffe-card__action) {
+                    text-decoration: underline;
+                }
             }
         }
     }
 
+    &:has(.ffe-card__action:visited) {
+        .native & {
+            @media (prefers-color-scheme: dark) {
+                color: var(--ffe-farge-hvit);
```

Comment:
Ser ikke ut som denne gjelder for det jeg ville trodd den gjør? 
![image](https://github.com/SpareBank1/designsystem/assets/155720728/a71a5637-414b-4e71-a320-9966bcec37c9)

Reply:
Sånn fikset!